Source/ExecutionEngine/CommandLineOptions.cs

```diff
@@ -142,11 +143,11 @@ protected virtual bool ParseOption(string name, CommandLineParseState ps)
     protected virtual string HelpHeader =>
       $"Usage: {ToolName} [ option ... ] [ filename ... ]" + @"
 
-    ---- General options -------------------------------------------------------
+  ---- General options -------------------------------------------------------
 
-    /version      print the " + ToolName + @" version number
-    /help         print this message
-    /attrHelp     print a message about supported declaration attributes";
+  /version      print the " + ToolName + @" version number
+  /help         print this message
+  /attrHelp     print a message about supported declaration attributes";
```

Comment:
Why remove the indent here?

Reply:
Did you comment on old changes? I don't see these changes in this PR

Reply:
Not sure why Github showed them to me.  Thanks!